OSBarcodeLib.podspec

```diff
@@ -0,0 +1,13 @@
+
+Pod::Spec.new do |spec|
+  spec.name = "OSBarcodeLib"
+  spec.version = "0.0.1"
```

Comment:
Considering we already have a released `1.0.0`, this should be set with that value.